charts/openshift-etcd-backup/Chart.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: openshift-etcd-backup
+description: A Helm chart for Openshift etcd backup configurations
```

Please include cluster type (=standalone) to better differentiate from the hcp etcd backup

```suggestion
description: A Helm chart for Standalone Openshift etcd backup configurations
```